app/models/description_change_request.rb

```diff
@@ -21,4 +23,9 @@ def rejected_reason_is_present?
   def set_previous_application_description
     self.previous_description = planning_application.description
   end
+
+  def set_sequence
+    change_requests = PlanningApplication.find(planning_application.id).description_change_requests
+    increment_sequence(change_requests)
+  end
```

Comment:
Could this repeated method not be pushed up into the parent ChangeRequest model?

Reply:
oh, I see, it's a different type each time. 🤔

Reply:
Would:
```
  def set_sequence
    self.sequence = self.where(planning_application: planning_application).count + 1
  end
```

work in ChangeRequest for all types?